evap/staff/tests/test_views.py

```diff
@@ -2280,6 +2280,57 @@ def test_exclude_unfinished_evaluations(self):
             # unfinished because still in EVALUATION_END_OFFSET_HOURS
             self.assertNotContains(page, self.evaluation2.full_name)
 
+    def test_suggested_evaluation_ordering(self):
+        evaluations = baker.make(
+            Evaluation,
+            course__semester=self.semester,
+            participants=[self.student1, self.student2],
+            voters=[self.student1, self.student2],
+            state=Evaluation.State.IN_EVALUATION,
+            vote_start_datetime=datetime.datetime.now() - datetime.timedelta(days=42),
+            vote_end_date=datetime.date.today() - datetime.timedelta(days=2),
+            can_publish_text_results=True,
+            _quantity=2,
+        )
+
+        for i, evaluation in enumerate(evaluations):
+            evaluation.general_contribution.questionnaires.set([baker.make(Questionnaire, type=Questionnaire.Type.TOP)])
+            question = baker.make(Question, type=Question.TEXT)
+            contribution = baker.make(
+                Contribution,
+                evaluation=evaluation,
+                contributor=baker.make(UserProfile),
+                questionnaires=[question.questionnaire],
+            )
+            baker.make(TextAnswer, contribution=contribution, question=question, answer=self.answer)
+            if i == 1:
+                baker.make(TextAnswer, contribution=contribution, question=question, answer=self.answer)
```

Comment:
Do I understand this right, that you simply want to add 1 TextAnswer for evaluations[0] and 2 TextAnswers for evaluations[1]? In this case:

```suggestion
        for evaluation, answer_count in zip(evaluations, [1, 2]):
            contribution = baker.make(Contribution, evaluation=evaluation, _fill_optional=["contributor"])
            baker.make(TextAnswer, contribution=contribution, question__type=Question.TEXT, _quantity=answer_count)
```